.circleci/config.yml

```diff
@@ -14,9 +14,9 @@ jobs:
       - run:
           name: lint
           command: npm run lint
-#      - run:
-#          name: test
```

Comment:
Looks like ci is failing because the file is not valid

Reply:
orps indentation, ok thanks